tests/checkout/klarna-paynow.spec.js

```diff
@@ -0,0 +1,23 @@
+// @ts-check
+const { test, expect } = require('@playwright/test');
+
+test('Klarna Pay Now', async ({ page }) => {
+
+    await page.goto('/');
+
+    await expect(page).toHaveTitle(/Checkout Demo/);
+    await expect(page.locator('text="Select a demo"')).toBeVisible();
+
+    // select Klarna Pay now
+    await page.locator('text="Klarna - Pay now"').click();
+    await expect(page.locator('text="Cart"')).toBeVisible();
+
+    // click "Continue to checkout"
+    await page.click('text="Continue to checkout"');
+    await expect(page.locator('text="Continue to Pay now with Klarna."')).toBeVisible();
+
+    // click "Continue to Klarna"
+    await page.locator('text="Continue to Pay now with Klarna."').click();
+    await expect(page).toHaveTitle("Je bestelling afronden");
+
```

Comment:
Let's keep comments-style consistent with the rest of the repo

``` 
// Select "Klarna Pay Now"
// Click "Continue to checkout"
// Click "Continue to Klarna"
```
Let's also remove the white-line on line 5 and 22 (nitpickiness at its finest haha ;)) 

This E2E will fail if you haven't enabled Klarna-pay in the CA. 

We're not actually making a payment, but this seems like a great to me for E2E!